From 1119596e55f70a1d6cac91ae290b31d9589e2f9f Mon Sep 17 00:00:00 2001
From: Patryk Kups <pkups@anaconda.com>
Date: Tue, 9 Dec 2025 10:42:03 +0100
Subject: [PATCH] patch

---
 pyproject.toml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 65a60eb..0a1b8dc 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -62,9 +62,6 @@ ml = [
 [tool.hatch.metadata]
 allow-direct-references = true
 
-[tool.hatch.metadata.hooks.custom]
-path = '../../scripts/hatch_hooks.py'
-
 [tool.hatch.build]
 directory = '../../dist/snowflake'
 dev-mode-dirs = ['src']
-- 
2.39.5 (Apple Git-154)

